content/zh/docs/tasks/administer-cluster/enabling-endpointslices.md

```diff
@@ -112,19 +114,48 @@ Kubernetes 中的 EndpointSlice 功能包含若干不同组件。它们中的大
   `WindowsEndpointSliceProxying`
   [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)。
 
+
+<!--
+## API fields
+
+Some fields in the EndpointSlice API are feature-gated.
+
+- The `EndpointSliceNodeName` feature gate controls access to the `nodeName`
+  field. This is an alpha feature that is disabled by default.
+- The `EndpointSliceTerminating` feature gate controls access to the `serving`
+  and `terminating` condition fields. This is an alpha feature that is disabled
+  by default.
+-->
+## API 字段
+
+EndpointSlice API 中的某些字段有对应的特性门控控制。
+
+- `EndpointSliceNodeName` 特性门控控制对 `nodeName` 字段的访问。这是默认情况下禁用的 Alpha 功能。
+- `EndpointSliceTerminating` 特性门控控制对 `serving` 和 `terminating` 状况字段的访问。这是默认情况下禁用的 Alpha 功能。
+
 <!--
 ## Using Endpoint Slices
 
 With EndpointSlices fully enabled in your cluster, you should see corresponding
 EndpointSlice resources for each Endpoints resource. In addition to supporting
-existing Endpoints functionality, EndpointSlices include new bits of information
-such as topology. They will allow for greater scalability and extensibility of
-network endpoints in your cluster.
+existing Endpoints functionality, EndpointSlices will allow for greater
+scalability and extensibility of network endpoints in your cluster.
 -->
 ## 使用 EndpointSlice
 
 在集群中完全启用 EndpointSlice 的情况下，你应该看到对应于每个
 Endpoints 资源的 EndpointSlice 资源。除了支持现有的 Endpoints 功能外，
-EndpointSlice 还引入了拓扑结构等新的信息。它们将使集群中网络端点具有更强的
-可伸缩性和可扩展性。
+EndpointSlices 将允许集群中网络端点更好的可伸缩性和可扩展性。
+
+<!--
+## {{% heading "whatsnext" %}}
+
+
+* Read about [EndpointSlices](/docs/concepts/services-networking/endpoint-slices/)
+* Read [Connecting Applications with Services](/docs/concepts/services-networking/connect-applications-service/)
+-->
+## {{% heading "whatsnext" %}}
+
 
+* 参阅 [EndpointSlices](/docs/concepts/services-networking/endpoint-slices/)
```

Comment:
add "/zh" prefix in links, like /zh/docs/concepts/services-networking/endpoint-slices/
same as below

Reply:
updated @howieyuen